addons/point_of_sale/static/src/js/models.js

```diff
@@ -1816,179 +1997,6 @@ exports.Orderline = Backbone.Model.extend({
 
         return taxes;
     },
-    /**
-     * Mirror JS method of:
-     * _compute_amount in addons/account/models/account.py
-     */
-    _compute_all: function(tax, base_amount, quantity, price_exclude) {
```

Comment:
Are you guys sure this is proper and as per guideline? and will not cause a problem for custom modules?
the Fact is this Commit is already landed into 13.0. and the modules which are inheriting/overriding even just calling _compute_all or compute_all method are now broken.